lib/service/servicecfg/alias.go

```diff
@@ -21,3 +21,7 @@ import "github.com/gravitational/teleport/lib/service"
 // Config is a temporary alias to aid in migrating configuration
 // from lib/service to lib/service/servicecfg.
 type Config = service.Config
+
```

Comment:
Mind leaving a TODO in the file hinting to a future cleanup?

Reply:
The future cleanup is coming _today_ so I'm not too worried about it.

Reply:
I thought you might say that,